src/clients/index.ts

```diff
@@ -1,3 +1,4 @@
 export * from './discord';
 export * from './express';
 export * from './astar';
+export * from './requestFilter';
```

Comment:
The name `requestFilter`  doesn't look like it should be in the `clients/` folder. What about changing this to something else, or moving the folder and treating it as a database client.